views/tx.tmpl

```diff
@@ -92,7 +86,11 @@
                         </div></td>
                         <!-- <td>{{$vin.Tree}}</td> -->
                         <td>
-                        {{if eq $vin.BlockHeight 0}}
+                        {{if $vin.Coinbase}}
+                            Created
+                        {{else if eq $vin.Txid "0000000000000000000000000000000000000000000000000000000000000000"}}
+                            Created
+                        {{else if eq $vin.BlockHeight 0}}
```

Comment:
since the above two are the coinbase and votereward cases, what is this third case for?

Reply:
unconfirmed regular transaction

Reply:
oh, but it shouldn't say created for the third case.  For that it should say N/A since it's just not there, at least not yet since the miner fills this in.  We should do it though in the near future, just not possible here.

Reply:
or maybe "pending" until we extract it manually

Reply:
Alright, I'm on it